Dockerfile

```diff
@@ -3,3 +3,13 @@ WORKDIR /usr/src
 ENV DEBIAN_FRONTEND=noninteractive
 RUN apt update -y
 RUN apt install -y curl cmake libgl1-mesa-dev libsdl2-dev libcurl4-openssl-dev libopenal-dev libfreetype6-dev mingw-w64 g++-mingw-w64 g++-multilib git zip vim-nox rsync
+COPY . /usr/src/
+
+RUN make -j
+
+ENTRYPOINT /usr/src/build/release-linux-x86_64/tremulous
```

Comment:
@erentar Thanks for the PR!

The usage and goal of this Dockerfile was to create only a building image for cross platform compilation. For the purpose of creating a "Runtime" image, you should build a new Docker image based on Ubuntu and pull the artifacts out of the buildtime image- this would reduce the size and scope of the final image.

The flow of this would look similar to the following

```Dockerfile
FROM ubuntu:bionic as tremulous_build
... 
RUN make -j

FROM ubuntu:bionic as tremulous_runtime
COPY --from=tremulous_build /usr/src/build/release-linux-x86_64 /usr/local/games/tremulous
ENTRYPOINT /usr/src/build/release-linux-x86_64/tremulous
...
```